pkgs/desktops/cinnamon/xapps/default.nix

```diff
@@ -0,0 +1,50 @@
+{ fetchFromGitHub, glib, gobject-introspection, gtk3, libgnomekbd, libxklavier, meson, ninja, pkgconfig, python3, python3Packages, stdenv, vala, wrapGAppsHook }:
+
+stdenv.mkDerivation rec {
+  pname = "xapps";
+  version = "1.6.4";
+  enableParallelBuilding = true;
+
+  src = fetchFromGitHub {
+    owner = "linuxmint";
+    repo = pname;
+    rev = version;
+    sha256 = "08c8j7wv56zf8mfgjfhd0wc1bbrw0dj1lmlawvsxwfajnnyjsm0d";
+  };
+
+  NIX_CFLAGS_COMPILE = [ "-I${glib.dev}/include/gio-unix-2.0" ]; # TODO: https://github.com/NixOS/nixpkgs/issues/36468
+
+  patches = [
+    ./py-override.patch # details see https://github.com/linuxmint/xapps/pull/65
+  ];
+
+  buildInputs = [ glib gtk3 libgnomekbd libxklavier pkgconfig python3Packages.pygobject3 vala ];
```

Comment:
`pkgconfig` and `vala` are not `buildInputs`, but `nativeBuildInputs`.

Reply:
Where is pygobject3 needed? I see a pygobject override, but that's just for consumers to use.

Reply:
I had to add pygobject3 package as native build input so pkgconfig could find it